DRList-1.0/DRList-1.0.lua

```diff
@@ -128,7 +130,14 @@ Lib.gameExpansion = ({
     [WOW_PROJECT_CLASSIC] = "classic",
     [WOW_PROJECT_BURNING_CRUSADE_CLASSIC or 5] = "tbc",
     [WOW_PROJECT_WRATH_CLASSIC or 11] = "wotlk",
-})[WOW_PROJECT_ID] or "wotlk" -- Fallback to wotlk when unknown (most likely a new classic expansion build)
+    [WOW_PROJECT_CATA_CLASSIC or 12] = "cata",
```

Comment:
The WOW_PROJECT_CATA_CLASSIC global exists now, but it's called WOW_PROJECT_CATACLYSM_CLASSIC and its value is 14.

![image](https://github.com/wardz/DRList-1.0/assets/16864855/c5f28166-f3cf-41a8-a02a-236f9f0413c6)